packages/react/src/hooks/useWallet.ts

```diff
@@ -63,7 +63,7 @@ export function useWallet(
   const { account } = useAccount();
 
   const _params: UseWalletParams =
-    typeof params === 'string' ? { account: params } : (params ?? {});
+    typeof params === 'string' ? { account: params } : params ?? {};
```

Comment:
could we revert this format change? i don't know why your biome insists on it haha